src/pages/[slug]/[ie]/index.tsx

```diff
@@ -593,7 +592,18 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 							onClickAction={onClickAction}
 						/>
 					</div>
+					{!mediaInfo?.description && (
+						<Alert
+							className="c-Alert__margin-bottom"
+							icon={<Icon name="info" />}
+							title={''}
```

Comment:
A11Y: maybe we can extend the alert component with a hideTitle option so we don't render an empty h-tag? 🤔 A temporary solution can be adding a className that hides the title using display none.

```css
.c-Alert__title--hidden {
    display: none;
}
```

Reply:
See https://github.com/viaacode/react-components/pull/73